docs/src/literate/quasi_incompressible_hyperelasticity.jl

```diff
@@ -0,0 +1,362 @@
+# # Nearly Incompressible Hyperelasticity
+#
+# ![](quasi_incompressible_hyperelasticity.gif)
+#-
+#md # !!! tip
+#md #     This example is also available as a Jupyter notebook:
+#md #     [`quasi_incompressible_hyperelasticity.ipynb`](@__NBVIEWER_ROOT_URL__/examples/quasi_incompressible_hyperelasticity.ipynb)
+#-
+# ## Introduction
+#
+# In this example we study quasi- or nearly-incompressible hyperelasticity using the stable Taylor-Hood approximation. In spirit, this example is the nonlinear analogue of 
+# [`incompressible_elasticity`](@__NBVIEWER_ROOT_URL__/examples/incompressible_elasticity.ipynb) and the incompressible analogue of
+# [`hyperelasticity`](@__NBVIEWER_ROOT_URL__/examples/hyperelasticity.ipynb). Much of the code therefore follows from the above two examples.
+# The problem is formulated in the undeformed or reference configuration with the displacement `u` and pressure `p` being the unknown fields. We now briefly outline
+# the formulation. Consider the standard hyperelasticity problem 
+#
+# ```math
+#   u = \argmin_{v\in\mathcal{K}}\Pi(v),\quad \text{where}\quad \Pi(v)  = \int_\Omega \Psi(v) d\Omega .
+# ```
+# For clarity of presentation we ignore any non-zero surface tractions and body forces and instead consider only
+# applied displacements (i.e. non-homogeneous dirichlet boundary conditions). Moreover we stick our attention to the standard Neo-Hookean stored energy density 
+# 
+# ```math
+#     \Psi(u) = \frac{\mu}{2}\left(I_1 - 3 \right) - \mu \log(J) + \frac{\lambda}{2}\left( J - 1\right){}^2.
+# ```
+# where I₁=F:F≡FᵢⱼFᵢⱼ and J = det(F) denote the standard invariants of the deformation gradient tensor F = 𝛪 + ∇u.
+# The above problem is ill-posed in the limit of incompressibility (or near-incompressibility), namely when
+# ```math
+#     \lambda/\mu \rightarrow +\infty
+# ```
+# In order to alleviate the problem, we consider the partial legendre transform of the strain energy density Ψ
```

Comment:
Are we really using a partial legendre transform here? If so, could you quickly give a hint on which variable it is applied to, since I still do not see it.

Edit: Maybe it is also helpful to leave a reference here for readers interested in the theory.

Reply:
Yes partial legendre transformation wrt J = det(F). I can provide a terse 3-line proof and the list references from where I learned the trick :). Thanks for the catch!!

Reply:
Done, now!